runtime/src/main/java/org/corfudb/runtime/Restore.java

```diff
@@ -154,6 +172,11 @@ private void restore() throws IOException {
         int restoredTableCount = 0;
         for (TableBackupFileName tableBackupFileName : tableBackupFileNames) {
             log.info("start restoring table {}", tableBackupFileName);
+            if (tableBackupFileName.namespace.equals(CORFU_SYSTEM_NAMESPACE) &&
+                    IGNORED_CORFU_SYSTEM_TABLES.contains(tableBackupFileName.getTableName())) {
+                log.debug("Skip restoring table {} which is part of IGNORED_TABLES", tableBackupFileName);
```

Comment:
nit: change to info level to match other logs in restore (this is important information IMO).